packages/react/src/components/ListBox/ListBoxSelection.tsx

```diff
@@ -74,7 +114,7 @@ function ListBoxSelection({
         className={`${prefix}--tag__close-icon`}
         onClick={handleOnClick}
         onKeyDown={handleOnKeyDown}
-        disabled={disabled}
+        // disabled={disabled} TODO determine if that's always been broken
```

Comment:
can we get more info on what's not working here? if there's an existing bug we can open a separate issue for this

Reply:
@francinelucca the error is:
```
Property 'disabled' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'
```

I guess `disabled` is not a correct property for a `<div/>`... :)

Reply:
should this be a `button` instead of a `div` with `role="button` 🤔, any thoughts here? The div doesn't seem to accept a disabled attribute indeed https://www.w3schools.com/tags/att_disabled.asp. any thoughts here @carbon-design-system/developers ?

Reply:
This seems related: https://github.com/facebook/react/issues/8977. I think it's safe to remove the `disabled` attribute entirely, given  `aria-disabled` is already set on this node. More thoughts welcome.

Reply:
@francinelucca I pushed a commit that removes the `disabled` attribute entirely. The goal of the typescript migration was to make as little changes as possible, so moving to `<button/>` is not the best idea here. Removing `disabled` attribute from that div is no harm given the link stated above, since it's not being respected anyway.

Reply:
@lewandom @francinelucca yes, can confirm we want to remove `disabled` here. Since the ListBox components are not exported and available to the consumer yet (#6177), removing this prop is not considered a breaking change.

Reply:
@tay1orjones since the changes for typescript were supposed to be minimal, I'd rather keep the component prop - it still has uses, e.g. for classes and event handling. I only removed setting the `disabled` attribute on the `div` element inside the component. Hope that's fine.